### ISSUE TITLE
Three small requests about preset training data

### ISSUE THREAD

**Issue by JoyfulGen** [DDMAL/Standalone-Interactive-Classifier#92]
My observations are currently based on the standalone IC; it's possible that Mothra IC behaves differently. Either way, my requests apply to both versions of the IC.

1. Currently, the Hufnagel training data is automatically checked when the page loads. I think none of the presets should be checked before the user checks them.
2. It's currently possible to check more than one preset training data, but I can't think of a situation where a user would want both square notation and hufnagel neumes. Could the behaviour be that checking one box unchecks the other boxes?
3. If I understood correctly, the Square2 training data was a test set. Could it be removed?
